komodo-core-tests/src/test/resources/org/komodo/repository/test/test-local-repository-on-file-config.json

```diff
@@ -84,6 +77,9 @@
         "removeDerivedContentWithOriginal" : true,
         "threadPool" : "modeshape-workers",
         "sequencers" : {
+            "VDB Dynamic Sequencer" : {
+				"classname" : "org.teiid.modeshape.sequencer.vdb.VdbDynamicSequencer"
```

Comment:
I added it to get the vdb.cnd loaded. It was needed in order for the tests to pass. Did you run the tests without it?